assignment/scripts/describe.js

```diff
@@ -164,6 +170,9 @@ if(minAge <= age) {
   console.log('no entry');
 } else {
   console.log('enter');
+// FIX line 170 and 172 should be swapped or swap the minAge and age variables. In current state this program compares minAge to age,
+//where we should be comparing age to minAge.
+//if (age <= minAge)
```

Comment:
Not quite. With this change, someone who is 21 gets "no entry" when they should get "entry".